docker-compose-cloud.buildx.yaml

```diff
@@ -35,7 +35,7 @@ services:
           - airbyte/server:${ALT_TAG:-${VERSION}}
         platforms:
           - linux/amd64
-          - linux/arm64          
+          - linux/arm64
```

Comment:
is this duplication needed?

Reply:
@benmoriceau Could you be more specific?  The two entries are different (amd64 vs arm64)